src/test/java/org/heigit/ohsome/ohsomeapi/controller/GetControllerTest.java

```diff
@@ -1,10 +1,8 @@
 package org.heigit.ohsome.ohsomeapi.controller;
 
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assume.assumeTrue;
-
```

Comment:
this newline should not have been removed, wtf

https://jenkins.ohsome.org/job/ohsome-api/view/change-requests/job/PR-100/14/checkstyle/fileName.1150936862/source.9c995717-0e4e-4de6-a98b-def3a2df83e7/#6

https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

Reply:
seems like the _organize imports_ feature of my IDE does not apply this -.-